server.js

```diff
@@ -142,6 +143,11 @@ function setEmotion(emotionName, init) {
   }
 }
 
+function restartReflectionAudio() {
+  // TODO: restart of reflection audio
```

Comment:
@lmccart I've left this for you for now because I think you're handling the sound / I don't currently have the Sonos setup or sound files, but can take a look if you'd like!